src/components/common/NameRequestInfo.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <div id="name-request-summary">
+    <v-row >
+      <v-col id="name-request-info" cols="4">
+        <ul>
+          <li> <label>
+          <strong>Name Request</strong>
+        </label></li>
+          <li>Entity Type: {{ entityTypeDescription() }}</li>
+          <li> Request Type: {{ requestType() }}</li>
+          <li>Expiry Date: {{ formattedExpirationDate() }}</li>
+          <li>
+            <v-icon v-if="nameRequestDetails.status === NameRequestStates.APPROVED" color="green"
+            class="nr-status-icon">mdi-check</v-icon>
+            Status: {{ formatStatusText(nameRequestDetails.status) }}
+          </li>
+          <li id="condition-consent">
+            <v-icon v-if="conditionConsent() === 'Received' || conditionConsent() === 'Not Required'"
+            color="green" class="nr-status-icon">mdi-check</v-icon>
+            <v-icon v-if="conditionConsent() === 'Not Received'"
+            color="red" class="nr-status-icon">mdi-close</v-icon>
+            Condition/Consent: {{ conditionConsent() }}
+          </li>
+        </ul>
+      </v-col>
+      <v-col id="name-request-applicant-info" cols="8">
+        <ul>
+          <li><label>
+          <strong>Name Request Applicant</strong>
+        </label></li>
+          <li>Name: {{ applicantName() }}</li>
+          <li>Address: {{ applicantAddress() }}</li>
+          <li>Email: {{ nameRequestApplicant.emailAddress }}</li>
+          <li>Phone: {{ nameRequestApplicant.phoneNumber | VMask('(###) ###-####')}}</li>
+        </ul>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Mixins, Vue, Prop } from 'vue-property-decorator'
+import { getName } from 'country-list'
+import { VueMaskFilter } from 'v-mask'
+
+// Enums
+import { NameRequestStates } from '@/enums'
+
+// Interfaces
+import { NameRequestIF, NameRequestDetailsIF, NameRequestApplicantIF } from '@/interfaces'
+
+// Mixins
+import { CommonMixin, DateMixin, NamexRequestMixin } from '@/mixins'
+
+@Component({
+  filters: { 'VMask': VueMaskFilter }
+})
+export default class NameRequestInfo extends Mixins(CommonMixin, DateMixin, NamexRequestMixin) {
+  // Template Enums
+  readonly NameRequestStates = NameRequestStates
+
+  @Prop()
+  private nameRequest: any
+
+  private parsedNameRequest: NameRequestIF
+
+  private nameRequestDetails:NameRequestDetailsIF
+
+  private nameRequestApplicant: NameRequestApplicantIF
+
+  private created () {
+    this.parsedNameRequest = this.generateNameRequestState(this.nameRequest, null)
+    this.nameRequestDetails = this.parsedNameRequest.details
+    this.nameRequestApplicant = this.parsedNameRequest.applicant
+  }
+
+  /** The entity title  */
+  private entityTypeDescription () :string {
```

Comment:
Use `enum-mixin.ts:entityTypeToName()` instead.

Reply:
The reason why I dint use that is the value is different. It returns 'BC Benefit Company' . Here the value is 'Benefit Company'

Reply:
Check with Scott. He probably want this consistent in both places.

Reply:
Checked with Scott. This was a missing in UX pin. Fixed the code to show the same data as it is in Create Business UI